frontend/public/components/storage-class.tsx

```diff
@@ -49,8 +45,6 @@ const StorageClassDetails: React.SFC<StorageClassDetailsProps> = ({obj}) => <Rea
       <dd>{obj.provisioner || '-'}</dd>
       <dt>Reclaim Policy</dt>
       <dd>{obj.reclaimPolicy || '-'}</dd>
-      <dt>Default Class</dt>
```

Comment:
It's not clear from the design docs if we want to move this part to the name of the storageClass, or keep it.

Reply:
Please take a look at the jira task created that I have linked above.